Telegram_bot/management/commands/main.py

```diff
@@ -34,32 +34,42 @@ def get_courses(message):
         text = ""
         for course in courses:
             course_name = f"{course['fullname']}"
-            button1 = types.InlineKeyboardButton(text=course_name, callback_data=f"{course_name} for_faq", resize_keyboard=True)
+            button1 = types.InlineKeyboardButton(text=course_name, callback_data=f"for_faq {course_name}", resize_keyboard=True)
             markup.add(button1)
             text += f"{course['fullname']} ({course['shortname']} {course['id']})\n"
         bot.send_message(message.chat.id, "Курсы на которые вы записаны: ", reply_markup=markup)
 
 
-@bot.callback_query_handler(func=lambda call: call.data.split()[1] == "for_faq")
-def info(call):
+@bot.callback_query_handler(func=lambda call: call.data.split()[0] == "for_faq")
```

Comment:
```suggestion
@bot.callback_query_handler(func=lambda call: call.data.split()[0] == "for_faq")
```
![image](https://github.com/ThreeAi/Telegram_bot/assets/102540775/d594945b-594f-430e-9278-39fcbf3a86f9)